packages/mdl-fab/mdl-fab.scss

```diff
@@ -92,6 +92,18 @@
     border: 0;
   }
 
+  /*
+    All direct children of the fab should have 100% width.
+    This allows for using SVGs within them to align properly in all browsers.
+  */
+
+  /* stylelint-disable selector-no-universal */
+
+  /* postcss-bem-linter: ignore */
+  > * {
```

Comment:
So if this is only happening for SVGs in Firefox (as discussed offline), we should probably limit this selector to SVGs and then put the specific reason why we're doing so into the comments.